### ISSUE TITLE
[Keyboard Navigation – Make Code - Review, Commit and push local changes to GitHub ]Focus order is not logical under screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3763]
User Experience:
End users who are dependent on the keyboard navigation might get affected and will not be able to navigate efficiently while navigating in the TAB mode if focus order is not logical on links tab.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Review, Commit and push local changes to GitHub" button and select it.
6. Navigate through all the controls appearing.
7. Verify the issue

Actual Result:
Focus order is not logical under screen using keyboard.

Expected Result:
Keyboard Focus order should be logical after pressing tab key.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[WCAG 2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG2.4.3; #Win10-Edge(Chromium);

![Uploading WCAG2.4.3.png…]()

**Comment by jwunderl** [comment]
No video with repro so maybe just out of date, but focus order seems correct to me - only behavior that is potentially different than one might expect would be from go back => pull changes, but it is moving in priority order / to most commonly used button first, then to the (much less commonly used, smaller) buttons.
